index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com"><link rel="preconnect" href="https://fonts.gstatic.com" crossorigin><link href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/7415ea4535.js" crossorigin="anonymous"></script>
+<body>
+<div class="container">
+    <header>
+        <h3 class="welcome"><a href="">Welcome</a></h3>
+        <nav>
+            <i class="fa-solid fa-bars small"></i>
+        </nav>
+    </header>
+
+    <section class="intro">
+        <h1>Hey there. I’m Mariane <br/><span>I’m a software developer</span></h1>
+        <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+    </section>
+
+    <div class="contacts">
+        <ul>
+            <li>
+                <a href="https://twitter.com/Stephen57913145"><i class="fa-brands fa-twitter"></i></a>
+            </li>
+            <li>
+                <a href="https://www.linkedin.com/in/stephen-katuli-a92752251/"><i class="fa-brands fa-linkedin"></i></a>
+            </li>
+            <li>
+                <a href=""><img src="/images/iconmonstr-medium-1.svg"></a>
+            </li>
+            <li>
+                <a href="https://github.com/stephenkati"><i class="fa-brands fa-github"></i></a>
+            </li>
+            <li>
+                <a href=""><i class="fa-brands fa-angellist"></i></a>
```

Comment:
- **[OPTIONAL]** All the images should have the `alt` attribute that describes the content of the image for a good practice of HTML&CSS, please 🙏🏽 add them.